test/unit/test_ovf.c

```diff
@@ -0,0 +1,125 @@
+#include <r_util.h>
+#include "minunit.h"
+
+int test_overflow_add(void) {
+	int a = 250;
+	int b = 32;
+	if (UT8_ADD_OVFCHK(a, b)) {
```

Comment:
these tests look wrong :/

Just do `mu_assert_false (UT8_ADD_OVFCHK (a, b), "ut8-add 1");`

Reply:
yeah i didnt liked this approach.ill change that now